src/selectors/entities/users.js

```diff
@@ -215,23 +229,23 @@ function sortAndInjectProfiles(profiles, profileSet, skipInactive = false) {
     return currentProfiles.sort(sortByUsername);
 }
 
-export const getProfiles = createSelector(
+export const getProfiles: (GlobalState, Filters) => Array<UserProfile> = createSelector(
     getUsers,
     (state, filters) => filters,
     (profiles, filters) => {
         return sortAndInjectProfiles(filterProfiles(profiles, filters), PROFILE_SET_ALL);
     }
 );
 
-function filterProfiles(profiles, filters) {
-    if (!filters || Object.keys(filters).length === 0) {
+function filterProfiles(profiles: IDMappedObjects<UserProfile>, filters: ?Filters): IDMappedObjects<UserProfile> {
```

Comment:
`filters` seems like an optional parameter, so `filters?: Filters` instead of `filters: ?Filters`?